.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
     - LEIN_GET='wget https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein'
     - LEIN_CHMOD='chmod +x ./lein'
     - LEIN_DOWN='lein downgrade 2.6.1'
+    - _JAVA_OPTIONS=''
```

Comment:
Can we add a comment about why this is needed?

Reply:
This change got removed?